Dockerfile

```diff
@@ -19,11 +19,10 @@ COPY controllers/ controllers/
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
 
-# Use distroless as minimal base image to package the manager binary
-# Refer to https://github.com/GoogleContainerTools/distroless for more details
 FROM bitnami/minideb:stretch
 WORKDIR /
 COPY --from=builder /workspace/manager .
+COPY third_party/licenses/licenses.csv /workspace/licenses.csv
```

Comment:
Note, you need to call `go-licenses save` to save the actual full text licenses into the image.
Use https://github.com/kubeflow/pipelines/blob/e58aff78f67b32d0ee2125b2678abf0c3fa4073a/v2/container/launcher/Dockerfile#L36-L37 as an example.

Reply:
thanks to your help. I've fix it.